### ISSUE TITLE
i got a error!!

### ISSUE THREAD

**Issue by TopSpoofer** [alexarchambault/coursier#359]
when i use coursier to get the 'hbase-hadoop2-compat', it down't work!
my builf.sbt look like:

```
scalacOptions ++= Seq(
  "-deprecation",
  "-feature",
  "-language:postfixOps",
  "-language:implicitConversions",
  "-language:reflectiveCalls"
)

val hbaseVersion = "1.1.1"
val hbaseHadoopCompat = "org.apache.hbase" % "hbase-hadoop-compat" % hbaseVersion % "compile"
val hbaseHadoop2Compat = "org.apache.hbase" % "hbase-hadoop2-compat" % hbaseVersion % "compile"
```

i got the request is : 
```
http://maven.aliyun.com/nexus/content/groups/public/org/apache/hbase/$%7Bcompat.module%7D/1.1.1/$%7Bcompat.module%7D-1.1.1.pom
[error]       http://repo.maven.apache.org/maven2/org/apache/hbase/$%7Bcompat.module%7D/1.1.1/$%7Bcompat.module%7D-1.1.1.pom
[error]       http://repo1.maven.apache.org/maven2/org/apache/hbase/$%7Bcompat.module%7D/1.1.1/$%7Bcompat.module%7D-1.1.1.pom
[error]       http://uk.maven.org/maven2/org/apache/hbase/$%7Bcompat.module%7D/1.1.1/$%7Bcompat.module%7D-1.1.1.pom

```
but not: 
```
http://uk.maven.org/maven2/org/apache/hbase/hbase-hadoop2-compat/1.1.1/hbase-hadoop2-compat-1.1.1.pom
```

how to solve this problem？
i found this: [error](https://www.bountysource.com/issues/36942039-can-not-resolve-hbase)

**Comment by benmccann** [comment]
This is potentially a duplicate of https://github.com/alexarchambault/coursier/issues/325 ?

**Comment by alexarchambault** [comment]
@TopSpoofer https://github.com/alexarchambault/coursier/issues/325 itself should be fixed (use version `1.0.0-M14-M6`).
